exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -1413,6 +1452,31 @@ public ColumnTypeMetadata_v3 getColumnTypeInfo(String[] name) {
       return directories;
     }
 
+    /** If directories list and file metadata list contain relative paths, update it to absolute ones
+     * @param baseDir base parent directory
+     */
+    @JsonIgnore public void updateRelativePaths(Path baseDir) {
```

Comment:
How is this method different from the previous one? Looks like they are doing a bit of the same thing...